package.json

```diff
@@ -11,6 +11,8 @@
     "build": "ng build",
     "dartanalyzer": "cd dist/dart && pub install && cd ../.. && ts-node scripts/ci/dart_analyzer",
     "demo-app": "cd src && ng serve",
+    "postinstall": "typings install  --ambient && npm prune",
+    "reinstall": "npm cache clean && rimraf coverage dist typings && rimraf node_modules && npm install",
```

Comment:
After discussing, keep the `postinstall`/`reinstall` scripts. When we do the publishing process we'll revisit this decision.

Replace `rimraf` (and remove the dependency) by `git clean -dfx`. This will remove everything that's not part of the git repository, even cached files for everything. This is okay for a `reinstall`, which is the same as rebuilding everything anyway. Alternatively, we could do a `git clean -dfx -e '.*'` (single quotes important) to keep dotfiles around (such as `.idea` and `.pub`).